projects/GKCore/GKCore/LangMan.cs

```diff
@@ -1596,7 +1596,7 @@ public static class LangMan
             /* 704 */ "Type of comparison",
             /* 705 */ "Internal comparing (search for duplicates)",
             /* 706 */ "Comparing with another database",
-            /* 707 */ "Analysis",
+            /* 707 */ "Analyze",

```

Comment:
This is a button, so imperative would be more appropriate.

Reply:
👍